src/bentoml_cli/models.py

```diff
@@ -254,7 +254,6 @@ def import_from(model_path: str) -> None:  # type: ignore (not accessed)
         "--context", type=click.STRING, default=None, help="Yatai context name."
     )
     @click.option(
-        "-f",
```

Comment:
Conflicting short option name with `--force`

Reply:
Let's just rename this to `-F`?